release.py

```diff
@@ -36,8 +35,9 @@ def release_github(test=True):
         if pace_ver != ver:
             raise Exception(f'version mismatch! __version__: {pace_ver}; {ver_name}: {ver}')
 
-    desc = re.search(r'# [v[0-9.]*[a-z0-9]{0,3}]\(http.*?\)\n(.*?)# [v[0-9.]*[a-z0-9]{0,3}]', changelog,
-                     re.DOTALL | re.MULTILINE).group(1).strip()
+    VERSION_PATTERN = r'v[0-9]+(\.[0-9]+){2}((a|b|rc)[0-9]+)?'
+    desc = re.search(rf'# \[{VERSION_PATTERN}\]\(http.*?\)\n(.*?)# \[{VERSION_PATTERN}\]',
+                    changelog, re.DOTALL | re.MULTILINE).group(4).strip()
```

Comment:
```suggestion
    VERSION_PATTERN = r'v[0-9]+(?:\.[0-9]+){2}(?:(?:a|b|rc)[0-9]+)?'
    desc = re.search(rf'# \[{VERSION_PATTERN}\]\(http.*?\)\n(.*?)# \[{VERSION_PATTERN}\]',
                    changelog, re.DOTALL | re.MULTILINE).group(1).strip()
```

Reply:
The ?: not capturing the group is very nice to know